specs/multispec.yml

```
@@ -33,6 +33,7 @@ specs:
       distros:
         - rhel-9-x86_64
       el_version: "9"
+      minimal_image: "ubi8/ubi-minimal:latest"
```

I guess RHEL 9 should not use UBI 8 minimal image.
```suggestion
      minimal_image: "ubi9/ubi-minimal:latest"
```